tests/test_tracking.py

```diff
@@ -29,29 +32,29 @@
 
 
 class TensorBoardTrackingTest(unittest.TestCase):
-    @classmethod
-    def teardown_class(cls):
-        shutil.rmtree("test_project_with_config")
-        shutil.rmtree("test_project_with_log")
-
     def test_init_trackers(self):
         hps = None
         project_name = "test_project_with_config"
-        accelerator = Accelerator(log_with="tensorboard")
-        config = {"num_iterations": 12, "learning_rate": 1e-2, "some_boolean": False, "some_string": "some_value"}
-        accelerator.init_trackers(project_name, config)
-        for child in Path(project_name).glob("*/**"):
-            log = list(filter(lambda x: x.is_file(), child.iterdir()))[0]
-            # The config log is stored one layer deeper in the logged directory
-            # And names are randomly generated each time
-        si = summary_iterator(str(log))
-        # Pull HPS through careful parsing
-        for event in si:
-            for value in event.summary.value:
-                proto_bytes = value.metadata.plugin_data.content
-                plugin_data = plugin_data_pb2.HParamsPluginData.FromString(proto_bytes)
-                if plugin_data.HasField("session_start_info"):
-                    hps = dict(plugin_data.session_start_info.hparams)
+        with tempfile.TemporaryDirectory() as dirpath:
+            oldpwd = os.getcwd()
+            os.chdir(dirpath)
```

Comment:
I'd like to avoid this. The API should accept any logging dir for TensorBoard at least, no?